### PR TITLE
🐛 Fix Homo sapiens casing

### PR DESCRIPTION
**Pull request by liberaliscomputing**
This PR changes/migrates the species enum value of human from `"Homo Sapiens"` to `"Homo sapiens"`.

### CODE REVIEW

**Review by fiendish — APPROVED** [review]
lgtm

File: migrations/versions/12850dab14e9_.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+"""Change 'Homo Sapiens' for 'Homo sapiens'
```
Comment (znatty22) [inline migrations/versions/12850dab14e9_.py]:
```suggestion
"""Change 'Homo Sapiens' to 'Homo sapiens'
```

File: migrations/versions/12850dab14e9_.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '12850dab14e9'
+down_revision = '025cf1ee0adf'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```
Comment (znatty22) [inline migrations/versions/12850dab14e9_.py:20]: Just curious, was this migration completely autogenerated from the change you made to the default for species? Or did you have to manually add the data migration code?

Comment (liberaliscomputing) [reply migrations/versions/12850dab14e9_.py:20]: I did `flask db revision` and a template file populated with the followings was created:

```
revision = '12850dab14e9'
down_revision = '025cf1ee0adf'
```

Comment (znatty22) [reply migrations/versions/12850dab14e9_.py:20]: Oh I mean did you have to modify what came out of `flask db migrate` at all?

Comment (liberaliscomputing) [reply migrations/versions/12850dab14e9_.py:20]: Oh, so I modified the following sections:

- module imports: https://github.com/kids-first/kf-api-dataservice/blob/fix-homo-sapiens-casing/migrations/versions/12850dab14e9_.py#L8-L9
- upgrade: https://github.com/kids-first/kf-api-dataservice/blob/fix-homo-sapiens-casing/migrations/versions/12850dab14e9_.py#L22-L29
- downgrade: https://github.com/kids-first/kf-api-dataservice/blob/fix-homo-sapiens-casing/migrations/versions/12850dab14e9_.py#L35-L42

Comment (fiendish) [reply migrations/versions/12850dab14e9_.py:20]: I don't think anything does data migrations automatically

**Pull request merged**